src/personaclient/PersonaClient.php

```diff
@@ -265,6 +273,36 @@ public function isPresignedUrlValid($url,$secret) {
         return $valid;
     }
 
+    /**
+     * Get a user profile based off a gupid passed in
+     * @param string $gupid
+     * @param string $token
+     * @access public
+     * @return mixed
+     * @throws \InvalidArgumentException
+     * @throws \Exception
+     */
+    public function getUserByGupid($gupid, $token){
+        if(trim($gupid) === '')
```

Comment:
Do i get anything by assigning the result back to the value? It's just doing some rudimentary validation at the moment and doesn't need to actually *do* anything with the value of ```$gupid```